_data/projects/OpinionatedGuides.yml

```
@@ -0,0 +1,11 @@
+name: Opinionated Guides
+desc: A one-stop guide to computers, linux, and electronics.
+site: https://github.com/opinionatedguide/opinionatedguide.github.io-src
+tags:
+- awesome
+- documentation
+- python
+- bash
+upforgrabs:
+    name: Hacktoberfest
```

Not sure how this happened, but looks like I added in whitespace unnecessarily, triggering that error

```suggestion
  name: Hacktoberfest
```

---

fixed